x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/job_details_step/components/advanced_section/components/model_plot/model_plot_switch.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useState, useContext, useEffect } from 'react';
+import { EuiSwitch } from '@elastic/eui';
+import { JobCreatorContext } from '../../../../../job_creator_context';
+import { isPopulationJobCreator } from '../../../../../../../common/job_creator';
+import { Description } from './description';
+
+export const ModelPlotSwitch: FC = () => {
```

Comment:
Will checks to display warnings if the model plot is enabled when there is a high of cardinality of fields be done in a follow up? I can't see a card in the GitHub project.

Reply:
Yes, this is a basic validation step and will be added in a follow up PR soon.